scripts/SANS/isis_reduction_steps.py

```diff
@@ -4282,7 +4282,7 @@ def execute(self, reducer, workspace):
             Reads the geometry information stored in the workspace
             but doesn't replace values that have been previously set
         """
-        _reducer = reducer  # noqa
+        _ = reducer  # noqa
```

Comment:
The class is derived from `ReductionStep` and overrides the `execute` method. See [here](https://github.com/mantidproject/mantid/blob/master/scripts/reducer_singleton.py#L31:L39). Changing the signature would of course be possible, but if the method is used via a kwarg, then it would break if we changed the argument signature here. Essentially we would be breaking the contract here.

On the other hand, I had a quick look and I can only find cases where the argument is used via its position, hence I would expect that the suggested change should not break anything in the current reduction setup. 

Do you think we should go ahead with this change?